src/breadcrumbs/_base.scss

```diff
@@ -2,40 +2,29 @@
  * @file Breadcrumbs
  * @module breadcrumbs
  * @overview Styles for Casper breadcrumbs
- * @TODO: expand/solidify styling, here they have styles to support grey and blue (like on the main site)
+ * @TODO: create set for footer breadcrumbs
```

Comment:
That was the original plan, but since @beckichoi said that the footer breadcrumbs would be more of a one off situation, it could make sense to name the `breadcrumbs--dark-background` `breadcrumbs--footer`?  I'd also rather make two separate breadcrumbs macros (footer/logo vs regular) but that seemed like more duplicate code.